benchmarks/template_benchmarks/template_compilation/benchmark.py

```diff
@@ -0,0 +1,57 @@
+from django import template
+
+from ...utils import bench_setup
+
+objects1 = [object(), object(), object(), object(), object()]
+objects2 = [object(), object(), object(), object(), object()]
+object1 = object()
+object2 = object()
+object3 = None
+num1 = 1
+num2 = 2
+boolean1 = True
+SCRIPT_CONTENT_URL = "/some/prefix"
+WEBSITE_DOMAIN = "http://www.somedomain.com"
+SHOW_ALT_HEADER = "True"
+
+
+class TemplateCompile:
+    def setup(self):
+        bench_setup()
+        self.context = {
+            "objects1": objects1,
```

Comment:
```suggestion
            "objects1": [object(), object(), object(), object(), object()],
```

We're not reusing these objects so I think we can create them as part of the setup.

Reply:
I have made the changes